tests/components/vesync/test_fan.py

```diff
@@ -32,6 +37,9 @@ async def test_fan_state(
     await hass.config_entries.async_setup(config_entry.entry_id)
     await hass.async_block_till_done()
 
+    coordinator = hass.data[DOMAIN][VS_COORDINATOR]
+    assert coordinator
```

Comment:
Please don't touch internals

Reply:
I was following patterns implemented in other integrations where data in hass.data[DOMAIN] is accessed and even this integration was already doing something like that [here](https://github.com/home-assistant/core/blob/55dc4b0d2c7450955e8d0cb37a0031aee61bed9b/tests/components/vesync/test_init.py#L57).

Reply:
Yep, we have quite some occurences in the code base, but we are trying to get them out slowly. So let's not try to add new ones